pkgs/games/gnonograms/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, vala
+, meson
+, ninja
+, pkg-config
+, desktop-file-utils
+, appstream
+, python3
+, wrapGAppsHook
+, gtk3
+, pantheon
+, libgee
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnonograms";
+  version = "1.4.5";
+
+  src = fetchFromGitHub {
+    owner = "jeremypw";
+    repo = "gnonograms";
+    rev = "v${version}";
+    sha256 = "1ly3inp6dvjrixdysz5hdfwlhbs49ks0lf8062z2iq6gaf8ivkb2";
+  };
+
+  # Don't update mime/desktop/icon/schemas caches
+  postPatch = ''
+    sed -i '/post_install/d' meson.build
```

Comment:
I am surprised it will run without schemas compiled.

Reply:
Typically, you would do `patchShebangs meson/post_install.py` in `postPatch` to be able to run the script.

Reply:
Oh? I thought wrapGAppsHook took care of that, the program does work for me.

I'll re-enable the script then.

Reply:
wrapGAppsHook only takes care of wrapping. Schema compilation is still responsibility of the package. This is discussed in https://github.com/NixOS/nixpkgs/issues/1455. Though, if you install it through environment.systemPackages, it [might be compiled by NixOS](https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/nixos/modules/config/system-path.nix#L170-L172).

Reply:
Looks like unlike most apps, the app just gracefully handles the cases when schema is not installed:

https://github.com/jeremypw/gnonograms/blob/18544494ef007392e36bb4e295500cd31ad4e70a/src/Controller.vala#L89-L91

Reply:
Thanks @jtojnar, this is good to know.
It should be fixed now.

Also wow, gsettings is _very_ complex..